executor/simple.go

```diff
@@ -136,6 +137,10 @@ func (e *SimpleExec) executeRollback(s *ast.RollbackStmt) error {
 	return nil
 }
 
+type sqlExec interface {
```

Comment:
Why do we need this interface? If it's necessary, it's better to put it to `util/sqlexec`.